src/materialized/src/http/templates/flamegraph.html

```diff
@@ -70,10 +70,28 @@ <h1>{{ title }}</h1>
 </div>
 
 <script>
+    function toHumanBytes(n) {
+        const tokens = ['B', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB'];
+        let iTok = 0;
+        while (Math.abs(n) >= 1024 && iTok < tokens.length - 1) {
+            n /= 1024;
+            ++iTok;
+        }
+        return n.toFixed(2) + " " + tokens[iTok];
+    }
+
     const chart = flamegraph()
         .width(960)
         .setDetailsElement(document.getElementById("details"))
-        .onClick(d => history.pushState({ id: d.id }, "", `#${d.id}`));
+        .onClick(d => history.pushState({ id: d.id }, "", `#${d.id}`))
+{% if display_bytes %}
+        .setLabelHandler(function (d) {
+            return d.data.name + ' (' + d3.format('.3f')(100 * (d.x1 - d.x0), 3) + '%, ' + toHumanBytes(d.value) + ' )';
+        })
+{% endif %}
+        ;
```

Comment:
nit: templating javascript is a bit mind bending, IMO. I like to extract into variables:

```js
const display_bytes = {{ display_bytes|json }};
let chart = /* ... */;
if displayBytes {
    chart = chart.setLabelHandler(d => {
        const percent = d3.format('.3f')(100 * (d.x1 - d.x0), 3);
        return `${d.data.name} (${percent}%, ${toHumanBytes(d.value)})`;
    });
}
```

Also, the backtick "template strings" can make things way more usable. They're supported in all modern browsers, and we're already using them on the line above.